llmfoundry/data/packing.py

```diff
@@ -447,6 +451,10 @@ def profile(raw_batch_size: int) -> Tuple[Optional[float], Optional[float]]:
         waste_percent = 100 * packer.waste
         return padding_percent, waste_percent
 
-    for packing_ratio, raw_batch_size in zip(packing_ratios, raw_batch_sizes):
-        padding, waste = profile(raw_batch_size)
-        yield (packing_ratio, padding, waste)
+    with logging_redirect_tqdm(loggers=[log]):
```

Comment:
Not sure this worked? At least I don't see any new tqdms in the logs for this search

Reply:
ah this one has the tqdm :) `log-test-qKA2k1`